VideoLocker/src/main/java/org/edx/mobile/util/Sha1Util.java

```diff
@@ -1,24 +1,28 @@
 package org.edx.mobile.util;
 
+import org.edx.mobile.logger.Logger;
+
 import java.io.UnsupportedEncodingException;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 
 public class Sha1Util {
+    private static final Logger logger = new Logger(Sha1Util.class);
 
     /**
-     * Returns SHA1 hash of the given text.
-     * @param text
-     * @return
-     * @throws NoSuchAlgorithmException
-     * @throws UnsupportedEncodingException
+     * @param text The plain text to encrypt.
+     * @return SHA1 hash of the given text or the plain text if encryption failed.
```

Comment:
Note that the 'encryption' phrase is present in a lot of places, including variables and method names. These should all be updated, I suppose.